client/src/pages/board/components/dashboardForms/CardDialog.js

```diff
@@ -0,0 +1,209 @@
+import React, {useReducer, useState, useCallback, useEffect} from 'react';
+import {
+  Box,
+  Dialog,
+  DialogActions,
+  Button,
+  Divider,
+  MuiThemeProvider,
+  CircularProgress,
+  makeStyles,
+  DialogContent,
+} from '@material-ui/core';
+import {MuiPickersUtilsProvider} from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import {getCardById} from 'api/Card';
+import CenteringBox from 'components/CenteringBox';
+import CardDialogTitle from './dialogSections/CardDialogTitle';
+import {dialogTheme} from '../../../../themes/theme';
```

Comment:
prob use the absolute imports here

Reply:
This is what I meant by "editor integration" in the previous discussion of absolute imports. VS Code has a setting to set preference of absolute vs relative imports but afaik, there is no way to say "use absolute imports if the path is too long and use relative imports if not". Which makes me question how useful/sustainable it is to maintain imports by hand.